### ISSUE TITLE
Mashup navigation - screenreader

### ISSUE THREAD

**Issue by vanch3d** [SAFeSEA/openEssayist-slim#9]
On ‘overview’ the T H P etc are read as “Structure: title empty region” and “Structure: heading empty region”. “Structure” seems to be repeated on all items so is perhaps redundant. “Empty region” will not mean anything to users.  It is possible to tab through the items, but as they are not links this would not be expected behaviour.
Ideally just “heading” or “discussion” should be announced. High priority (because currently so confusing).
